CHANGES.md

```diff
@@ -23,10 +23,16 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
+ * The list command will only display provider based implementations by default.
+   The -backwards-compatibility must be specified to display algorithm
+   implementations that exist for legacy and engine purposes.
```

Comment:
Well... That's not the best behavior, I think. 

If no `propquery` is provided, I think, the `list` command should enumerate all the available algorithms. 
If `propquery` is provided, iy should be applied (and skip all the engine-provided algorithms).

Reply:
And done.